tests/plugins/balloonpanel/cssloading.js

```diff
@@ -17,13 +17,13 @@
 				// Our release version is built with moono-lisa skin inlined, thus we can't
 				// test it against other skin. We don't have straight way to recognise editor's
 				// built version, such trick must be used (#1251).
-				'test loading css with path': CKEDITOR.revision !== '%REV%'
+				'test loading css with path': bender.tools.isBuild()
 			}
 		},
 
 		'test loading css with path': function() {
-			assert.areSame( 'kama,/apps/ckeditor/skins/kama/', CKEDITOR.skinName, 'config skinName should be with path' );
-			sinon.assert.calledWith( spy, balloonTestsTools.getDocumentOrigin() + '/apps/ckeditor/plugins/balloonpanel/skins/kama/balloonpanel.css' );
+			assert.areSame( 'kama,%BASE_PATH%../../apps/ckeditor/skins/kama/', CKEDITOR.skinName, 'config skinName should be with path' );
```

Comment:
Please add a short note of explanation to why `removeDottsFromUrl` function is used, and add an issue reference ticket related to this PR.